cluster/etcd.go

```diff
@@ -68,6 +68,16 @@ func (c *Cluster) DeployRestoreCerts(ctx context.Context, clusterCerts map[strin
 }
 
 func (c *Cluster) DeployStateFile(ctx context.Context, stateFilePath, snapshotName string) error {
+	stateFileExists, err := util.IsFileExists(stateFilePath)
```

Comment:
Does the `DeployStateFile` only get called when creating etcd snapshot?

Reply:
Yes, but in case of multiple calls this would be the same requirement as we can't deploy it if the state file doesnt exist/cant be read locally

Reply:
Ok, I just want to make sure we are giving the right information. If it gets called elsewhere we have to change the warning message accrodingly.